MATLAB/Utilities/ParkerWeight.m

```diff
@@ -15,14 +15,16 @@
 %   publisher={American Association of Physicists in Medicine}
 % }
 if unique(geo.DSD)>1

```

Comment:
There is a big here that was there in my code. This should be `if length(unique(geo.DSD))>1`. Could you add that?